package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": "true",
   "description": "Node schedule utilities for use with Renovate bot.",
-  "main": "index.js",
+  "main": "built/src/index.js",
```

```suggestion
  "main": "build/src/index.js",
```

I think this needs to change as well?

---

Yes, I missed that one. Good catch.